api/releases/v1.8.0-rc.toml

```diff
@@ -0,0 +1,17 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+sub_path = "api"
+ignore_deps = [ "github.com/containerd/containerd" ]
+
+# previous release
+previous = "v1.7.0"
```

Comment:
q: This will be `api/v*` from next release onwards right?

Reply:
Yes